Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -133,7 +133,7 @@ def run
         result.sub!("class #{class_name} < Formula", "class #{versioned_name} < Formula")
 
         # Remove bottle blocks, as they won't work.
-        result.sub!(BOTTLE_BLOCK_REGEX, "")
+        result.sub!(BOTTLE_BLOCK_REGEX, "# The bottles might be available as oci images at docker://ghcr.io/homebrew/core/#{name}:#{version}")
```

Comment:
What if, instead of injecting a comment, we output a warning saying "bottles have been removed because XYZ, but they may still be available at this URL if you'd like to extract them". That way the URL is displayed for easy copy/paste without needing to open the new file to find it, and it also doesn't pollute the new file

Reply:
This output will be lost with the end of the terminal session while comment stays with the formula forever. And it's easier to script. Perhaps we should do both?